install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -295,12 +295,16 @@ data:
   # container image names
   sidecar-istio-proxy-image: "{{ .Values.proxy.sidecarImageRegex }}"
 
+{{- if not .Values.gke.enabled }}
   # Encapsulation mode for communication between nodes
   # Possible values:
   #   - disabled
   #   - vxlan (default)
   #   - geneve
+  #
+  # In case gke is enabled, the tunnel key is set at line 319.
```

Comment:
Please avoid line numbers, this is very likely to get inconsistent in the future. I'd just use “below in this file”.

Reply:
Got it!! XD

Reply:
Thanks! Can I be annoying? I think adding this comment was in fact a mistake, I didn't remember that it would be added to files `quick-install.yaml` and `experimental-install.yaml`, and it doesn't make sense to have it in those files. We should probably remove this line, I'm terribly sorry for the u-turn and misleading you on this :(.

Reply:
No problem :D